webserver/script/LoadForecasting/UploadToPostgres.py

```diff
@@ -160,27 +163,34 @@ def run(
         work_control
     ):
 
+        # replaces the oldest profile when profile count = 4
+        profile_count = LoadForecastConfig.objects.count()
+        if (profile_count >= 4):
```

Comment:
Pull the `4` into a constant or a variable set on instantiation via a defaulting param. Right now the 4 is just a magic random number.

Reply:
thanks for pointing out